src/managed/Microsoft.Extensions.DependencyModel/Microsoft.Extensions.DependencyModel.csproj

```diff
@@ -4,6 +4,7 @@
   <PropertyGroup>
     <Description>Abstractions for reading `.deps` files.</Description>
     <TargetFrameworks>net451;netstandard1.3;netstandard1.6</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard1.3;netstandard1.6</TargetFrameworks>
```

Comment:
I tried your change on VS 2017 Update 2 and building/intellisense seems to work fine.  I wasn't able to get Test Explorer to work, but that obviously has nothing to do with what you're doing here.

So I think this is a good approach.

---
In reply to: [116104351](https://github.com/dotnet/core-setup/pull/2370#discussion_r116104351) [](ancestors = 116104351)